src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/additions/Addition.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.additions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.ChangeListItem;
+
+public abstract class Addition extends ChangeListItem {
+    @JsonProperty("kind")
+    private String kind;
```

Comment:
What kind is this supposed to be --
       Remember we have an overall Filing "kind" so this might be confusing with that kind ?
       The overall Filing Kind will be "overseas-entity-update" a fixed constant.

Reply:
So kind in this case is in reference to the 'kind' of addition it is as defined in the ODP

```
{ 
    change: “AddOfficerAppointment”,
    kind: “corporate-managing-officer”,
    officerAppointment: { }
}
```

To make it a bit more clear I can change the name to **additionKind** in the Java model (but keep it as **kind** in the JSON output)

Reply:
For complete clarity, the 'kind' property in Addition has been changed to 'type'